Makefile

```diff
@@ -11,7 +11,8 @@ endif
 develop: install-python-dependencies setup-git
 
 setup-git:
-	pre-commit install
+	pip install 'pre-commit==2.4.0'
```

Comment:
What is special about this version?

Reply:
Nothing special, it's just the latest. And pinning to something just helps reproducibility down the line.

FWIW I've observed no noticeable difference in the past between pre-commit 1.x and 2.x.